tiatoolbox/dataloader/slide_info.py

```diff
@@ -0,0 +1,50 @@
+"""
+This file contains code to output or save slide information using python multiprocessing
+"""
+from tiatoolbox.dataloader import wsireader
+from tiatoolbox.decorators.multiproc import TIAMultiProcess
+
+import os
+
+
+@TIAMultiProcess(iter_on="input_path")
+def slide_info(input_path, output_dir=None, mode="show"):
+    """
+    Single file run to output or save WSI meta data. Multiprocessing uses this function to run slide_info in parallel
+    Args:
+        input_path: Path to whole slide image
+        output_dir: Path to output directory to save the output
+        mode: "show" to display meta information only or "save" to save the meta information
+
+    Returns:
+        displays or saves WSI meta information
+
```

Comment:
Blank space at end of docstring. I know we have mentioned this before, but what did we decide on? I think it looks more tidy without the trailing blank space

Reply:
There will be a blank space. I might agree with you that the code looks more tidy but may be Google style doesn't 😊
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html